nextcloud_install_production.sh

```diff
@@ -241,7 +215,7 @@ CREATE USER $NCUSER WITH PASSWORD '$PGDB_PASS';
 CREATE DATABASE nextcloud_db WITH OWNER $NCUSER TEMPLATE template0 ENCODING 'UTF8';
 END
 print_text_in_color "$ICyan" "PostgreSQL password: $PGDB_PASS"
-service postgresql restart
+systemctl restart postgresql
```

Comment:
is here a postgresql.service missing? you have done it this way below.

Reply:
Yes, I will fix all the details later, thanks for pointing it out. :)